### DISCUSSION TITLE
Better than koodo reader ！！！

### DISCUSSION THREAD

**Discussion by samchanart** [mienaiyami/yomikiru#327] [Show and tell]
This reader is better than koodo reader for reading e-books. Thank you very much, I hope that in the future to support the setting interface multilingual.

**Comment by mienaiyami** [comment]
Thank you.

> I hope that in the future to support the setting interface multilingual.

Unfortunately, I can't do that now. It will only be possible when this app gets popular enough so native speakers can contribute to translation.

**Comment by zeoint** [comment]
For me, its the best all in one reader. I don't see any other reader replacing this one at all even in the future.

This works great, the scrolling is very smooth, its low cpu and memory footprint, the user interface is to the point without any unnecessary options, offline, and this offers the best and great reading experience.

(Those library managers are a mess, be it interface, cpu and memory footprint, and reader experience, they all are awful.)

**Reply by omodia** [reply]
True.
